registry/DeploymentTemplates/azuredeploy.json

```diff
@@ -142,7 +142,8 @@
         "subnetRef": "[concat(variables('vnetId'), '/subnets/default')]",
         "tenantId": "[subscription().tenantId]",
         "location": "[resourceGroup().location]",
-        "provisionScriptParameters": "[concat('ADMIN_USER_NAME=', parameters('adminUsername'),' SA_RESOURCE_ID=', parameters('storageAccountResourceId'),' SA_CONTAINER=', parameters('storageAccountContainer'),' KV_RESOURCE_ID=', parameters('pfxKeyVaultResourceId'),' CERT_THUMBPRINT=', parameters('pfxThumbprint'),' PIP_LABEL=', parameters('pipDomainNameLabel'),' REGISTRY_TAG=', parameters('registryTag'),' SPN_CLIENT_ID=',parameters('servicePrincipalClientId'),' SPN_CLIENT_SECRET=',parameters('servicePrincipalClientSecret'),' ENABLE_VALIDATIONS=',parameters('enableValidations'),' REGISTRY_REPLICAS=', parameters('registryReplicas'))]"
+        "marketplaceVersion": "1.0.0",

```

Comment:
this is risky, it can get out of sync

Reply:
This I just added for now so that we done need to update the script any more. I Ideally I want to put as param. This I am holding till we have a good CI run.